corehq/apps/prototype/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import re_path as url
+from corehq.apps.prototype.views import example
+
+urlpatterns = [
+    url(r'^example/$', example.knockout_pagination,
```

Comment:
Hi Biyeun, should I be able to open something locally by going to "http://localhost:8000/prototype/example/"? Or no, because there is no view?

Reply:
yes, you can go to it

Reply:
<img width="561" alt="image" src="https://github.com/dimagi/commcare-hq/assets/39149002/ddaa9b82-8ab5-4b45-8387-79ee807820c8">
I saw this when I go to "http://localhost:8000/prototype/example/". Did I missed anything?